src/omv/ports/stm32/sensor.c

```diff
@@ -421,6 +426,18 @@ static void mdma_memcpy(vbuffer_t *buffer, void *dst, void *src, int bpp, bool t
 }
 #endif
 
+#if (OMV_ENABLE_SENSOR_MDMA_TOTAL_OFFLOAD == 1)
```

Comment:
I don't think this is needed anymore, it should by default fully offload to MDMA.